gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/LogsServiceImpl.java

```diff
@@ -379,7 +379,14 @@ private String getSubscription(io.gravitee.repository.log.model.ExtendedLog log)
             try {
                 ApiKeyEntity key = apiKeyService.findByKeyAndApi(log.getSecurityToken(), log.getApi());
                 if (key != null) {
-                    return key.getSubscription();
+                    // TODO
```

Comment:
`TODO` comment should be removed.

Could we simplify it ? maybe by adding `Optional<SubscriptionEntity> ApiKeyEntity.getSubscriptionForApi(api)` ?